.github/workflows/testing_changes.yml

```
@@ -221,28 +224,84 @@ jobs:
           args: -vv --no-default-features --features '${{ matrix.features }}'
           # Searching for fbclient.lib in rsfbclient-native/build.rs
 
-      - name: Run cargo test
+      - name: Run cargo test on rsfbclient
         uses: actions-rs/cargo@v1
         if: matrix.plataform != 'macos' || matrix.build != 'linking'
         with:
           command: test
-          args: --no-default-features --features '${{ matrix.features }}' --workspace -- --test-threads 1
+          args: --no-default-features --features '${{ matrix.features }}' -- --test-threads 1
+
+      - name: Run cargo test on rsfbclient-rust
+        uses: actions-rs/cargo@v1
+        if: matrix.plataform != 'macos' || matrix.build != 'linking'
+        with:
+          command: test
+          args: -p rsfbclient-rust
+
+      - name: Run cargo test on rsfbclient-diesel
+        uses: actions-rs/cargo@v1
+        if: matrix.plataform != 'macos' || matrix.build != 'linking'
+        with:
+          command: test
+          args: --manifest-path rsfbclient-diesel/Cargo.toml --no-default-features --features '${{ matrix.features_diesel }}'' -- --test-threads 1
```

```suggestion
          args: --manifest-path rsfbclient-diesel/Cargo.toml --no-default-features --features '${{ matrix.features_diesel }}' -- --test-threads 1
```

btw string literals in workflows are supposed to be single quotes
https://docs.github.com/en/free-pro-team@latest/actions/reference/context-and-expression-syntax-for-github-actions#literals

but if it's working fine right now i'm ok not to touch the rest of the file

---

Fixed